spec/index.bs

```diff
@@ -1741,13 +1738,43 @@ enum GPUAddressMode {
 };
 </script>
 
+<dl dfn-type="enum-value" dfn-for=GPUAddressMode>
+    : <dfn>"clamp-to-edge"</dfn>
+    ::
+        Texture coordinates are clampled between 0.0 and 1.0, inclusive.
```

Comment:
These will likely need a ton of expanding once we get to specifying how graphics operations work but look like a good initial description.

Reply:
Oh yes, this isn't exactly correct (see the comment on `GPUAddressMode`)